openapi.yaml

```
@@ -559,8 +559,12 @@ components:
               ownerAddress:
                 type: string
                 example: '0x6EC0DEeD30605Bcd19342f3c30201DB263291589'
-                description: Domain recipient wallet address. Email or owner could be provided in order to detect domain receiver. If owner is provided - domain will be minted to the owner address. If nothing is provided, the domain will be minted to the reseller's internal custodial wallet.
+                description: Domain recipient wallet address. Email or owner could be provided in order to detect domain receiver. If owner is provided - domain will be minted to the owner address. If ownerAddress is not provided and useCustodialWallet = true, the domain will be minted to the reseller's custodial wallet.
```

I don't think we need to replicate whatever is written in the endpoint description here. So I suggest minimalistic description here:

```suggestion
                description:  Owner wallet address a domain will be minted for. Can not be specified together with email or useCustodialWallet.
```